src/components/FilterSearch.tsx

```diff
@@ -51,6 +51,15 @@ export interface FilterSearchProps {
   placeholder?: string,
   /** Whether to trigger a search when an option is selected. Defaults to false. */
   searchOnSelect?: boolean,
+  /** A function which is called when a filter is selected. To use this, searchOnSelect must be false. */
```

Comment:
can `searchOnSelect` be undefined which is falsey or does it have to be `false`? If falsey is ok, can you change to "_To use this, searchOnSelect must not be true_"

Reply:
now that `searchOnSelect` is being marked as deprecated and it's value doesn't affect `onSelect`, I removed this line from the docs